src/cf-expandables/src/ExpandableTransition.js

```diff
@@ -98,8 +98,14 @@ function ExpandableTransition( element ) {
   function expand() {
     this.dispatchEvent( 'expandBegin', { target: this } );
 
-    if ( !previousHeight || element.scrollHeight > previousHeight ) {
-      previousHeight = element.scrollHeight;
+    let childrenHeight = 0;
+    for ( const child of element.children ) {
```

Comment:
No IE support for `for … of`. Do we care? It may also be picked up by babel.

Reply:
Ah, right. I'll change it to `Array.prototype.forEach.call`.